server/src/infra/entities/asset.entity.ts

```diff
@@ -148,6 +148,16 @@ export class AssetEntity {
 
   @OneToMany(() => AssetFaceEntity, (assetFace) => assetFace.asset)
   faces!: AssetFaceEntity[];
+
+  @Column({ type: 'varchar', nullable: true })
+  stackParentId?: string | null;
+
+  @ManyToOne(() => AssetEntity, (asset) => asset.stack, { nullable: true })
+  @JoinColumn({ name: 'stackParentId' })
+  stackParent?: AssetEntity | null;
+
+  @OneToMany(() => AssetEntity, (asset) => asset.stackParent, { onDelete: 'SET NULL' })
```

Comment:
It doesn't look like this `onDelete SET NULL` made it into the migration. Can you also add `onUpdate: CASCADE` as well though. I think these settings might been to be in the `stackParent` relation. It means "when the parent is deleted, set the child's `stackParentId` to `NULL`. The one you have here means "when a child is deleted...", which doesn't make sense. Try moving it an re-generating the migration. Also, since there is no set null on delete you shouldn't be able to execute a parent asset delete in the database, which should continue to work.

Reply:
True. My bad. Made the required changes now. Thanks a lot.